src/lib/badges/Badge.svelte

```diff
@@ -31,36 +29,91 @@
     ['!pink']: 'bg-pink-500 text-pink-100'
   };
 
-  const hovers = {
-    blue: 'hover:bg-blue-200 dark:hover:bg-blue-300',
-    dark: 'hover:bg-gray-200 dark:hover:bg-gray-300',
-    red: 'hover:bg-red-200 dark:hover:bg-red-300',
-    green: 'hover:bg-green-200 dark:hover:bg-green-300',
-    yellow: 'hover:bg-yellow-200 dark:hover:bg-yellow-300',
-    indigo: 'hover:bg-indigo-200 dark:hover:bg-indigo-300',
-    purple: 'hover:bg-purple-200 dark:hover:bg-purple-300',
-    pink: 'hover:bg-pink-200 dark:hover:bg-pink-300'
+  const borderedColors = {
+    blue: 'bg-blue-100 text-blue-800 dark:bg-gray-700 dark:text-blue-400 border-blue-400',
+    dark: 'bg-gray-100 text-gray-800 dark:bg-gray-700 dark:text-gray-400 border-gray-500',
+    red: 'bg-red-100 text-red-800 dark:bg-gray-700 dark:text-red-400 border-red-400',
+    green: 'bg-green-100 text-green-800 dark:bg-gray-700 dark:text-green-400 border-green-400',
+    yellow: 'bg-yellow-100 text-yellow-800 dark:bg-gray-700 dark:text-yellow-300 border-yellow-300',
+    indigo: 'bg-indigo-100 text-indigo-800 dark:bg-gray-700 dark:text-indigo-400 border-indigo-400',
+    purple: 'bg-purple-100 text-purple-800 dark:bg-gray-700 dark:text-purple-400 border-purple-400',
+    pink: 'bg-pink-100 text-pink-800 dark:bg-gray-700 dark:text-pink-400 border-pink-400'
+  };
+
+  const hoverColors = {
+    blue: 'hover:bg-blue-200',
+    dark: 'hover:bg-gray-200',
+    red: 'hover:bg-red-200',
+    green: 'hover:bg-green-200',
+    yellow: 'hover:bg-yellow-200',
+    indigo: 'hover:bg-indigo-200',
+    purple: 'hover:bg-purple-200',
+    pink: 'hover:bg-pink-200'
   };
 
+  const baseClass: string = 'font-medium inline-flex items-center justify-center';
+
   let badgeClass: string;
   $: badgeClass = classNames(
     baseClass,
-    large ? 'text-sm font-medium' : 'text-xs font-semibold',
-    colors[color],
-    href && (hovers[color] ?? hovers.blue),
-    rounded ? 'rounded-full p-1' : 'rounded px-2.5 py-0.5',
+    large ? 'text-sm px-2 py-1' : 'text-xs px-2.5 py-0.5',
```

Comment:
I've rechecked with Flowbite.com and currently they use `px-2.5 py-05` for all badges regardless the size.